File: kyu-7/_0026__Growth_of_a_Population/solution.js
function nbYear(p0, percent, aug, p) {
  let year = 0;
  while (p0 < p) {
    p0 += Math.floor((p0 * percent) / 100 + aug);
    year += 1;
  }
  return year;
}

console.log(nbYear(1500, 5, 100, 5000));

File: kyu-7/_0026__Growth_of_a_Population/solution.rb
def nb_year(p0, percent, aug, p, age=0)
  while p0 < p
    p0 += (p0 * percent / 100 + aug).to_i
    age += 1
  end 
  age
end

p nb_year(1500, 5, 100, 5000)